### PR TITLE
Correct wording to use "SLE BCI" consistently

### PR DESCRIPTION
**Pull request by dirkmueller**
SUSE BCI, SLE-BCI, BCI should all be standardized to SLE BCI.
- Leaving out the FAQ as that is going to be moved to suse.com.
- Fixing Busybox -> BusyBox typo

### CODE REVIEW

File: content/guides/adding-users.adoc [hunk 1]
```diff
@@ -1,34 +1,34 @@
 ---
-title: Adding Users to BCI Micro and Minimal
+title: Adding Users to SLE BCI Micro and Minimal
 ---
 
 {{< hint type="note" >}}
-This guide will demonstrate how to add users to the BCI Micro and Minimal
+This guide will demonstrate how to add users to the SLE BCI Micro and SLE Minimal
```
Comment (dcermak) [inline content/guides/adding-users.adoc:6]: This looks wrong, I'd say either "and SLE BCI Minimal" or just "Minimal", but not "SLE Minimal"

File: content/guides/adding-users.adoc [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 manager in your final image and additionally:
 
 1. you do not need `rpm`
 
 2. your application runs with POSIX `sh` and not just Bash
 
-then consider using the BCI Busybox image instead. It is even smaller than BCI
-Micro and ships the Busybox implementation of `useradd`. Adding a new user in
-Busybox is straightforward:
+then consider using the SLE BCI BusyBox image instead. It is even smaller than BCI
```
Comment (dcermak) [inline content/guides/adding-users.adoc:29]:
```suggestion
then consider using the SLE BCI BusyBox image instead. It is even smaller than SLE BCI
```

File: content/guides/adding-users.adoc [hunk 3]
```diff
@@ -102,25 +102,25 @@ USER $user
 ----
 
 Build your container image using your favorite container runtime using the
-`--build-arg` parameter as in link:#_Switch_to_using_the_Busybox_BCI[Switch to
-using the Busybox BCI].
+`--build-arg` parameter as in link:#_Switch_to_using_the_BusyBox_BCI[Switch to
```
Comment (dcermak) [inline content/guides/adding-users.adoc:105]: This link is now wrong

**Review by dcermak — CHANGES_REQUESTED** [review]
Looks mostly good to me, except for a few tiny issues.

File: content/guides/adding-users.adoc (hunk 1 above)
Comment (dirkmueller) [reply content/guides/adding-users.adoc:6]: resolved

File: content/guides/adding-users.adoc (hunk 3 above)
Comment (dirkmueller) [reply content/guides/adding-users.adoc:105]: corrected

File: content/guides/adding-users.adoc [hunk 4]
```diff
@@ -102,25 +102,25 @@ USER $user
 ----
 
 Build your container image using your favorite container runtime using the
-`--build-arg` parameter as in link:#_Switch_to_using_the_Busybox_BCI[Switch to
-using the Busybox BCI].
+`--build-arg` parameter as in link:#_Switch_to_using_the_SLE_BCI[Switch to
```
Comment (dcermak) [inline content/guides/adding-users.adoc:105]:
```suggestion
`--build-arg` parameter as in link:#_switch_to_using_the_busybox_sle_bci[Switch to
```

**Review by dcermak — CHANGES_REQUESTED** [review]
The link is still not working.

**Review by dcermak — APPROVED** [review]
LGTM, thanks!

**Pull request merged**